lib/vendor/advancedvideo.js

```diff
@@ -0,0 +1,74 @@
+(function () {
+	var videoAdvanced;
+
+	videoAdvanced = function (element, {url, fallback, frameRate = 29.997, frameNum, source}) {
```

Comment:
You unfortunately can't use destructuring (or any ES6 features) here because files in `vendor/` are not compiled.

Alternatively we may want to move this into `showImages.js` or its own file outside of `vendor/`, although we don't have a good story for the latter right now.